spec/models/hathi_links_spec.rb

```diff
@@ -3,101 +3,89 @@
-                                       url: 'https://hdl.handle.net/2027/12345',
-                                       additional_text: nil,
-                                       etas_item: false
-                                     })
+        document = { 'oclc_number_ssim': ['12345'],
+                     'ht_access_ss': 'allow' }
+        hathi_link = SolrDocument.new(document).hathi_links
+
+        expect(hathi_link).to match({
+                                      text: I18n.t('blackcat.hathitrust.public_domain_text'),
+                                      url: 'https://catalog.hathitrust.org/api/volumes/oclc/12345.html',
+                                      additional_text: nil,
+                                      etas_item: false
+                                    })
       end
-    end
-
-    context 'with etas mono item' do
-      let(:document) { { 'hathitrust_struct': ['{"ht_id":"12345","access":"deny"}'] } }
 
       it 'generates a url to the checkout page or scan view with the etas text' do
-        expect(hathi_links).to match({
-                                       text: I18n.t('blackcat.hathitrust.etas_text'),
-                                       url: 'https://hdl.handle.net/2027/12345?urlappend=%3Bsignon=swle:urn:mace:incommon:psu.edu',
-                                       additional_text: I18n.t('blackcat.hathitrust.etas_additional_text'),
-                                       etas_item: true
-                                     })
+        document = { 'oclc_number_ssim': ['12345'],
+                     'ht_access_ss': 'deny' }
+        hathi_link = SolrDocument.new(document).hathi_links
+
+        expect(hathi_link).to match({
+                                      text: I18n.t('blackcat.hathitrust.etas_text'),
+                                      url: 'https://catalog.hathitrust.org/api/volumes/oclc/12345.html?urlappend=%3Bsignon=swle:urn:mace:incommon:psu.edu',
+                                      additional_text: I18n.t('blackcat.hathitrust.etas_additional_text'),
+                                      etas_item: true
+                                    })
       end
     end
 
-    context 'with etas multi/serial item' do
-      let(:document) { { 'hathitrust_struct': ['{"ht_bib_key":"12345","access":"deny"}'] } }
-
-      it 'generates a url to the catalog record with the etas text' do
-        expect(hathi_links).to match({
-                                       text: I18n.t('blackcat.hathitrust.etas_text'),
-                                       url: 'https://catalog.hathitrust.org/Record/12345?urlappend=%3Bsignon=swle:urn:mace:incommon:psu.edu',
-                                       additional_text: I18n.t('blackcat.hathitrust.etas_additional_text'),
-                                       etas_item: true
-                                     })
-      end
+    it 'generates a url to the catalog record with the etas text' do
+      document = { 'oclc_number_ssim': ['12345'],
```

Comment:
do we need this spec?

Reply:
seems like similar to this spec https://github.com/psu-libraries/psulib_blacklight/pull/679/files?file-filters%5B%5D=.erb&file-filters%5B%5D=.rb&file-filters%5B%5D=.xml&w=1#diff-2f43653230e5e7a28792e91d306131d398675e5ce5051ecefd53496a7d12b774R19

Reply:
🤦